e2e/tests/functional/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -32,9 +32,19 @@ const path = require('path');
 const NOTEBOOK_NAME = 'Notebook';
 
 test.describe('Notebook CRUD Operations', () => {
-  test.fixme('Can create a Notebook Object', async ({ page }) => {
+  test.beforeEach(async ({ page }) => {
+    //Navigate to baseURL
+    await page.goto('./', { waitUntil: 'domcontentloaded' });
+  });
+  test('Can create a Notebook Object', async ({ page }) => {
     //Create domain object
+    await createDomainObjectWithDefaults(page, {
+      type: NOTEBOOK_NAME
+    });
     //Newly created notebook should have one Section and one page, 'Unnamed Section'/'Unnamed Page'
+    const notebookSectionNames = page.locator('.c-notebook__sections .c-list__item__name');
+    await expect(notebookSectionNames).toBeHidden();
+    await expect(page.locator('span').filter({ hasText: 'Unnamed Page' })).toBeVisible();
```

Comment:
Let's do the same here. Assert on the visibility and text of both the section and page names in the Notebook's header

Reply:
@ozyx Thanks for your patience and help with the pull request. It's the newly created notebook part that I am struggling with. I will make the changes tomorrow.

Reply:
My pleasure! You've done a fantastic job so far. We are aware there is a bit of a learning curve not only with Playwright but also with Open MCT itself. Take as much time as you need, there's no rush. 

As always, let us know if you have any questions-- we are here to help. And thank you for your hard work to improve Open MCT!